src/main/java/com/bynder/sdk/service/impl/BynderServiceImpl.java

```diff
@@ -51,18 +52,25 @@
      * Instance of the collection service.
      */
     private CollectionService collectionService;
+    /**
+     * options for the http connection to Bynder.
+     */
+    private HttpConnectionSettings httpConnectionSettings;
+
+
 
     /**
      * Initialises a new instance of the class.
      *
      * @param baseUrl Base URL where we want to point the API calls.
      * @param credentials Credentials to use to call the API.
+     * @param httpConnectionSettings settings for the http connection to Bynder
      */
-    private BynderServiceImpl(final URL baseUrl, final Credentials credentials) {
+    private BynderServiceImpl(final URL baseUrl, final Credentials credentials, final HttpConnectionSettings httpConnectionSettings) {
```

Comment:
create a new constructor instead with this new argument

Reply:
now I have two constructors, but the old one is never used